From 2c0dc79882a788968fab2fb7304040b5ab46c6d3 Mon Sep 17 00:00:00 2001
From: karanpat <karan.patidar@intel.com>
Date: Tue, 5 Feb 2019 21:11:58 +0530
Subject: [PATCH] Disable safemedia feature for cel_apl

Default volume for SafeMediaVolumeDevices is restricted to 3. Bypass safemedia.

Tracked-On:
Signed-off-by: karanpat <karan.patidar@intel.com>
---
 groups/audio/project-celadon/init.rc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/groups/audio/project-celadon/init.rc b/groups/audio/project-celadon/init.rc
index 406d39a..d0e4a9d 100644
--- a/groups/audio/project-celadon/init.rc
+++ b/groups/audio/project-celadon/init.rc
@@ -1,3 +1,5 @@
 on boot
     insmod /vendor/lib/modules/kernel/sound/drivers/snd-dummy.ko
 
+on post-fs
+    setprop audio.safemedia.bypass true
-- 
2.20.1

